package.json

```diff
@@ -13,8 +13,9 @@
     "build.test": "gulp build.test --color",
     "build.test.watch": "gulp build.test.watch --color",
     "generate.manifest": "gulp generate.manifest --color",
-    "e2e": "protractor",
-    "e2e.live": "protractor --elementExplorer",
+    "e2e": "protractor src/browser/protractor.conf.js",
```

Comment:
Should we then also drop the whole `"dependencies": { ... }` section in the root `package.json`, since this has moved to the `src/browser/package.json` ?